### PR TITLE
feat: use core constants

### PR DESCRIPTION
**Pull request by negebauer**
This first needs notify-watcher/core#7 to be merged and released as new version.
Then updating the version of `notify-watcher/core` here

### CODE REVIEW

File: gtd/config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  description: 'GTD Manquehue plans watcher',
+  auth: false,
+  libs: ['axios', 'cheerio', 'lodash'],
+  timeframe: {
+    type: TIMEFRAMES.day,
+    hour: 10,
+  },
+  notificationTypes: {
+    newPlan: {
+      key: 'newPlan',
```
Comment (sasalatart) [inline gtd/config.js]: Should these have display names as well?

Comment (sasalatart) [reply gtd/config.js]: (A description could probably be a long paragraph...)

Comment (negebauer) [reply gtd/config.js]: yeah they should, will leave an issue for that

Comment (negebauer) [reply gtd/config.js]: _some_ should, others don't need it

Comment (negebauer) [reply gtd/config.js]: opened #26 for this

**Pull request merged**